docker-compose.test.yml

```diff
@@ -0,0 +1,27 @@
+version: '2'
```

Comment:
Do you need this docker-compose? are you using it somewhere?

Reply:
I am testing, why checks are queued